### PR TITLE
Jetty 9.4.x optimize parser warning from logs and add spotbug

### PR DESCRIPTION
**Pull request by olamy**
current logs parser are reading multiple time log files so this generate extra load on jenkins server.
this generate aggregated report per jdk as well (more readable)
Screen Shot 2020-11-27 at 6 57 47 pm

### CODE REVIEW

**Review by joakime — APPROVED** [review]
+1

**Comment by sbordet** [comment]
@olamy why is the `-T2` option back? Did not we agree to disable it and have the build doing its course?

File: Jenkinsfile [hunk 1]
```diff
@@ -12,8 +12,7 @@ pipeline {
           steps {
             container('jetty-build') {
               timeout( time: 240, unit: 'MINUTES' ) {
-                mavenBuild( "jdk8", "clean install", "maven3",
-                            [[parserName: 'Maven'], [parserName: 'Java']])
+                mavenBuild( "jdk8", "-T2 clean install", "maven3")
```
Comment (joakime) [inline Jenkinsfile]: I agree with @sbordet, lets remove the `-T2` here.

File: Jenkinsfile [hunk 2]
```diff
@@ -42,8 +42,8 @@ pipeline {
           steps {
             container( 'jetty-build' ) {
               timeout( time: 240, unit: 'MINUTES' ) {
-                mavenBuild( "jdk11", "clean install -Djacoco.skip=true", "maven3",
-                            [[parserName: 'Maven'], [parserName: 'Java']])
+                mavenBuild( "jdk11", "-T2 clean install -Djacoco.skip=true -Perrorprone", "maven3")
```
Comment (joakime) [inline Jenkinsfile]: I agree with @sbordet, lets remove the `-T2` here.

File: Jenkinsfile [hunk 3]
```diff
@@ -54,8 +54,8 @@ pipeline {
           steps {
             container( 'jetty-build' ) {
               timeout( time: 240, unit: 'MINUTES' ) {
-                mavenBuild( "jdk15", "clean install -Djacoco.skip=true", "maven3",
-                            [[parserName: 'Maven'], [parserName: 'Java']])
+                mavenBuild( "jdk15", "-T2 clean install -Djacoco.skip=true", "maven3")
```
Comment (joakime) [inline Jenkinsfile]: I agree with @sbordet, lets remove the `-T2` here.

File: Jenkinsfile [hunk 4]
```diff
@@ -78,9 +79,8 @@ pipeline {
           agent { node { label 'linux' } }
           steps {
             container( 'jetty-build' ) {
-              timeout( time: 60, unit: 'MINUTES' ) {
-                mavenBuild( "jdk8", "-Pcompact3 clean install -DskipTests", "maven3",
-                            [[parserName: 'Maven'], [parserName: 'Java']])
+              timeout( time: 120, unit: 'MINUTES' ) {
+                mavenBuild( "jdk8", "-T3 -Pcompact3 clean install -DskipTests", "maven3")
```
Comment (joakime) [inline Jenkinsfile]: I agree with @sbordet, lets remove the `-T3` here.

**Review by joakime — CHANGES_REQUESTED** [review]
Changing my review.
Lets drop the maven `-T#` arguments.

**Review by joakime — APPROVED** [review]
LGTM.

**Pull request merged**